.github/workflows/ci.yml

```
@@ -0,0 +1,62 @@
+name: GitHub CI
+
+on:
+  - pull_request
+  - push
```

This is what you can do if you do not want Actions to run on feature branches (another option is to only open PRs from forks):

```suggestion
on:
  push:
    branches:
      - v8.13
  pull_request:
    branches:
      - '**'
```

---

can you match on branches `v*` or something like that (and master)?